highfidelity/hifi
# Merged interstitial bug fixes

- Merged fixes for bugs in 0.75.0 for the interstitial page

## TEST PLAN (OPTIMIZATION PR #14240)
you really only need to confirm that one domain works at all... because this change will be a "it works" or "it doesn't work" -- it should be really obvious that it's working.

## TEST PLAN (INTERSTITIAL LOADING PR #14200)
1) Launch interface
2) Launch netlimiter
3) Reduce bandwidth to 10Mbps (bits not bytes)
4) Travel to large domains like Ozone
5) The loading bar should be less stagnate in most domains.
Note: If its taking a long to download the last few entities the loading bar will not progress and appear stagnate.

## TEST PLAN (TELEPORT PR #14188)
Steps to Reproduce: Unknown  -below is an estimated guess.
1) Log into an account,
2) Log out, then back into a second player account.
3) Change domain, Move away from spawn by using the warp.
4) Repeat 3 and 2 for as many time you wish.
5) Interstitial bubble should not be displaced.

## TEST PLAN (WEB ENTITY PR #14180)
1) In VR, go to a domain which causes the interstitial loading screen to appear for at least a few seconds
2) Place a web entity in front of the spawn point
3) Go to a different domain
4) Return to the first domain
5) Attempt to click the "Go To TheSpot" abort button
6) Laser should not intersect with placed web entity and you should be able to use the button.
7) the web entities should be intersect-able when the interstitial screen goes away.

## TEST PLAN (IGNORE ENTITY PR #14161)
- Launch interface in HMD mode
- Go to a domain that has entities blocking the abort button (i.e. hifi://gigs)
- Try to hover over the abort button without clicking it
- Land in attemped domain (i.e. hifi://gigs)
- Entity originally attempted should be clickable with lasers
